R/read_extract_delayed_discharges.R

```diff
@@ -0,0 +1,18 @@
+#' Read Delayed Discharges extract
+#'
+#' @param year Year of BOXI extract
+#'
+#' @return zsav/rds data file for Delayed discharges
+#' @export
+#'
+read_extract_delayed_discharges <- function(year) {
```

Comment:
`year` is a redundant parameter here, did you mean to use `dd_period` which could be passed to `get_dd_path` ?
```suggestion
#' @return zsav/rds data file for Delayed discharges
#' @export
#'
read_extract_delayed_discharges <- function() {
```

Reply:
Good spot! Checked `dd_period` and we don't need this because `get_dd_path` comes from the raw data with all years. When the year is supplied in `process_extract_delayed_discharges` this will use the function `is_date_in_fyyear` to correctly filter out episodes within the year